src/smarteiffel/generation/eiffeldoc/eiffeldoc_globals.e

```diff
@@ -233,6 +234,11 @@ feature {}
          html_os.close_title
          html_os.put_stylesheet(css)
          html_os.put_javascript(js)
+         html_os.open_tag(once "noscript")
```

Comment:
I guess this may be found guilty of talking to a low-level interface in a high-level class. Should I add methods creating `<noscript>` and `<style>` elements to `HTML_OUTPUT_STREAM` instead and call them here?